addons/mail/static/src/models/user_setting/user_setting.js

```diff
@@ -281,5 +281,12 @@ registerModel({
         voiceActiveDuration: attr({
             default: 0,
         }),
+        /**
+         * Models that represent the volume chosen by the user for each partner.
```

Comment:
The old string is kinda bad though.

suggestion -> Determines the volume chosen by the current user for each other user.